docs/tutorials/register.md

```diff
@@ -90,14 +90,16 @@ print(docsutils.fig_to_html(fig)) # markdown-exec: hide
 ```
 
 !!! warning "Units for qubit coordinates"
-    Qubits coordinates in Qadence are *dimensionless* but converted to the required unit when executed on a backend.
-	For instance, [Pulser](https://github.com/pasqal-io/Pulser) uses $\mu \textrm{m}$.
+    In general, Qadence makes no assumption about the units given to qubit coordinates.
+    However, if used in the context of a Hamiltonian coefficient, the quantity $Ht$
+    must be **dimensionless** for exponentiation in the PyQTorch backend, where it is assumed that $\hbar = 1$.
```

Comment:
```suggestion
    must be **dimensionless** for exponentiation in the PyQTorch backend, where it is assumed that $\hbar = 1$ (also called atomic units).
```

Reply:
I think I would rather not call them atomic units because we are not dealing with any other atomic related constants, just setting hbar to 1. Natural unit systems can be very general and arbitrary anyway, and Hartree atomic units is just one possible system. I'm sure there are others that also set hbar to 1.

Reply:
OK. Related to the above, H.t has [pi rad] as phase unit right ?

Reply:
If `hbar = 1` then `H.t` needs to be dimensionless so it can indeed represent an "angle" in the exponential, which you could write as "rads", but radians are not physical units.

Reply:
I've kept `rad` in the units of the docs for now because they do it in Pulser but I would be in favour of eventually removing that.